manuscript.Rmd

```diff
@@ -1165,6 +1165,29 @@ This analysis plan may contain data-dependent decisions to increase flexibility
 
 ## Limitations
 
+We have to distinguish two kinds of limitations.
+First, some limitations are by design and are unlikely to be changed soon.
+Our workflow does inherit these from the software it relies on and the fundamental design principles these share with the workflow and `repro`.
+Second, there are other limitations that we hope development effort of the open-source community and, concerning `repro`, us will ease.
+
+As a principle, this workflow is incompatible with steps that cannot be automated.
+This principle is at odds with some ingrained habits of researchers to mix and match manual and automated steps in data analysis.
+To allow for automation, many researchers will have to search for alternative software.  
```

Comment:
Researchers may need to adopt free open source software  to allow for automation.

Also: Consider citing papers that have made the case that we need to make this shift anyway. Have a look at this paper (Deepl translate if necessary): https://decorrespondent.nl/12477/zolang-het-onderwijs-geen-eigen-apps-bouwt-danst-het-naar-de-pijpen-van-google-en-microsoft/383742612-2efe26be?fbclid=IwAR2vIdQIsbLqSUg0MOCXLXdSeWNcNdH6QohbbR_kZ1Nx3kSjz08KTOxgt5Q